.src/c_adapter.cpp

```diff
@@ -4,7 +4,7 @@
 
 #include <iostream>
 
-#include <.src/c_adapter.hpp>
+#include "c_adapter.hpp"
```

Comment:
Maybe I'm misunderstanding the purpose of replacing <> with "" everywhere, but the reason I intentionally kept the `.src/` and `include/` prefix was, so that when a user types in their IDE something like:
```
#include <
``` 
then I don't want my .src files showing up in the recommended completions (the thingy where when you press tab it auto-completes the statement). This is especially true with c_adapter, I don't want any of the users to even be aware what functions are in there, as it can break things in very odd ways. All files in .src and the entire c_adapter should be as hidden from end-users as possible. 
Idk if this is true, but it feels like your restructuring made it, so users would have an easier time accessing the source and inline files, which I don't like. Correct me if I am wrong, though, as I'm not super sure what the full reason for the switch to "" was

Reply:
In gcc, clang, (and I think msvc) `"` based brackets search the current directory of the current file before searching the global include path.  Where as `<` doesn't search locally first.

I moved them because they have to be installed some where to build a downstream library based on libjluna.  When using a package manager, you can't assume that you have the cloned repo available, only the files explicitly installed.

Unfortunately, even with the `<` style includes, I still get auto complete suggestions for things in `.src`  it's kind of unavoidable.  There otherway to hide them, but it doesn't for me.

In the case where I don't want user's touching things, I typically use the PIMPL idiom to hide the implemenation and leave the details in the shared object as much as possible.  Unfortunately that can be difficult with highly templatized code like this.  I can't wait for wide support for modules when this can be a thing of the past.